seed/accounts/account.py

```
@@ -0,0 +1,39 @@
+from dependency_injector.wiring import Provide, inject
+
+from mpt_api_client.resources.accounts.account import Account
+from seed.container import Container
+from seed.context import Context
+from seed.helper import init_resource
+
+logger = logging.getLogger(__name__)
+
+
+@inject
+async def init_vendor_account_id(context: Context = Provide[Container.context]) -> Account:  # noqa: RUF029
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `RUF029`)

Remove unused `noqa` directive

(RUF100)